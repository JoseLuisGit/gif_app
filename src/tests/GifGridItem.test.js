import { shallow } from 'enzyme';
import { GifGridItem } from "../components/GifGridItem";


describe('TestComponent ', ()=>{

    let wrapper ;
     
    let value = {
        id: 'alosy78fw',
        title: 'my title',
        img: 'content_img'
    } 

    beforeEach(()=>{
       wrapper = shallow(
           <GifGridItem 
            { ...value }
           />
       );
    })
        test('visualizar componente', ()=>{
            expect( wrapper ).toMatchSnapshot();
        });

})