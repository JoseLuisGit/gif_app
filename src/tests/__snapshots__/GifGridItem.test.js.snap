// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent  visualizar componente 1`] = `
<div
  className="card"
>
  <img
    alt="my title"
  />
  <p>
    my title
  </p>
</div>
`;
